acceptance/openstack/autoscaling/v1/configurations_test.go

```diff
@@ -19,7 +19,7 @@ func TestConfigurationsList(t *testing.T) {
 	configs, err := configurations.List(client, listOpts)
 	th.AssertNoErr(t, err)
 
```

Comment:
Could you add `policies` tests for v1? `Instances` and `quotas` tests would be great too.